### PR TITLE
Delete `test/REQUIRE` and fix tests

### PR DESCRIPTION
**Pull request by DilumAluthge**
This file is no longer necessary.

### CODE REVIEW

**Comment by DilumAluthge** [comment]
Bump @nalimilan

**Comment by nalimilan** [comment]
Looks good, but tests fail for an obscure dependencies incompatibility issue...

**Comment by DilumAluthge** [comment]
@nalimilan Alright I believe I fixed the issue by tweaking the General registry. I think this PR is ready to merge now.

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,12 @@ julia:
   - 1.0.4
   - 1.1
   - 1.2
+  - 1.3
   - nightly
 
 matrix:
   allow_failures:
-    - julia: nightly
+    # - julia: nightly
```
Comment (nalimilan) [inline .travis.yml]: I'd rather keep this as failure can happen later and it's annoying to have to change it back.

**Comment by nalimilan** [comment]
Any idea about the Documenter failure?

**Pull request merged**